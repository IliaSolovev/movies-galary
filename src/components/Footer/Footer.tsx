import React from 'react';

import { Logo } from '..';

import style from './footer.module.scss';

export const Footer: React.FC = () => (
  <footer className={style.footer}>
    <Logo />
  </footer>
);
