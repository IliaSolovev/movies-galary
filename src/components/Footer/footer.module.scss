@import './../../styles/variables';
@import './../../styles/main';

.footer{
  display: flex;
  align-items: center;
  height: 5vh;
  background: rgba(0, 0, 0, 0.7);
  margin-bottom: 2vw;
  font-family: "Roboto",sans-serif;
  font-size: 1.2em;
  font-weight: bold;
  color: $redColor;
  & > *{
    @extend .container;

  }
}