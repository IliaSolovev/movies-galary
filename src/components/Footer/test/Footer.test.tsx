import React from "react";
import {shallow} from "enzyme";
import Footer from "../Footer";
import toJson from "enzyme-to-json";
it('should render footer', () => {
    const component = shallow(<Footer/>);
    expect(toJson(component)).toMatchSnapshot();
});