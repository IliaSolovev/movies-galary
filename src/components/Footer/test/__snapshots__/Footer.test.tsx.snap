// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render footer 1`] = `
<footer
  className="footer"
>
  <Component />
</footer>
`;
